tests/tests.js

```diff
@@ -49,8 +49,16 @@ QUnit.test("put text here", function(assert) {
 
 /* --- ExtractDataModule.movieDB --- */
 
-QUnit.test("put text here", function(assert) {
-    assert.equal(1, 1, "text goes here")
+QUnit.test('Test getActorId function', function(assert) {
+    var result = ExtractDataModule.movieDB.getActorId(mockData.actor1.results[0]);
```

Comment:
should only pass the argument mockData.actor1 as that is the response we will give it